src/server/controllers/survey.controller.js

```diff
@@ -0,0 +1,90 @@
+import mongoose from 'mongoose';
```

Comment:
There are no unit tests for any of these methods. Are they coming in a separate PR?